openerp/workflow/service.py

```diff
@@ -68,6 +69,15 @@ def delete(self):
         WorkflowInstance(self.session, self.record, {}).delete()
 
     def create(self):
+
+        def check_module(module):
+            """Check if the module is loaded"""
+            return sys.modules.get('openerp.addons.%s' % module[0])
+
+        def to_record(module_record):
+            """Return the resource id"""
+            return (module_record[1], )
```

Comment:
Could you add `:param type module:`, `:param type module:` and `:returns:` elements to the doc. Namely, explain why you're fetching a certain index.